index.js

```diff
@@ -325,6 +326,40 @@ class Story extends Base {
   }
 }
 
+class CardShareStory extends Story {
+  constructor(config, story, card) {
+    super(config, story);
+    this.card = card;
+  }
+
+  ogAttributes() {
+    var url = this.url + '/card/' + this.card.id;
+    var storyOgAttributes = super.ogAttributes();
+    var obj = !this.card ? {} : {
+      "title": _.get(this.card, ['metadata', 'social-share', 'title'], this.story["headline"]),
+      "description": _.get(this.card, ['metadata', 'social-share', 'message'], this.story["summary"]),
+      "image": _.get(this.card, ['metadata', 'social-share', 'image', 'url'], `${this.config['cdn-name']}/${_.get(this.card, ['metadata', 'social-share', 'image', 'key'])}`.replace(" ", "%20")),
```

Comment:
We can just generate the image url from s3 key instead of image-url because that comes from cloud front.